.github/workflows/jenkins.yaml

```diff
@@ -0,0 +1,48 @@
+name: Galaxy Test Suites
+on: [push, pull_request]
+env: 
+  GALAXY_TEST_DBURI: 'postgres://postgres:postgres@localhost:5432/galaxy?client_encoding=utf8'
+jobs:
+
+  test:
+    name: Test
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        test-suite: ['integration']
+        python-version: [3.7]
+    services:
+      postgres:
+        image: postgres:11
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: postgres
+        ports:
+          - 5432:5432
+    steps:
+    - name: Setup Minikube
+      if: matrix.test-suite == 'integration'
+      id: minikube
+      uses: CodingNagger/minikube-setup-action@v1.0.2
+    - name: Launch Minikube
+      if: matrix.test-suite == 'integration'
+      run: eval ${{ steps.minikube.outputs.launcher }}
+    - name: Check pods
+      if: matrix.test-suite == 'integration'
+      run: |
+        kubectl get pods
+    - name: Cache venv dir
+      uses: actions/cache@v1
+      id: cache-venv-galaxy
+      with:
+        path: .venv
```

Comment:
Should we cache both `~/.cache/pip` and `.venv` ? When this will be used for multiple galaxy branches, different dependencies will be installed in the virtualenv.

Reply:
But then you would still have a cache miss in~/.cache/pip, I'd expect this wouldn't reduce test setup time. We could possibly key on the target branch, but I don't think there would be any benefit to caching both locations

Reply:
> But then you would still have a cache miss in `~/.cache/pip`

Why? If you try `pip install -v somepackage==someversion` with alternate package versions, you'll see that pip will use cached data.

Reply:
How is that possible ? You only have one version in the cache, right ?

Reply:
I mean the way the github cache works, I understand if there was some kind of merge happening when uploading the cache, but if the cache was hit it won't be updated.

Reply:
Ah, I didn't know that the `actions/cache` for GitHub workflows works differently than the Travis cache (where the cache is updated at every successful build).

Anyway, it seems there are separate caches for different branches https://help.github.com/en/actions/automating-your-workflow-with-github-actions/caching-dependencies-to-speed-up-workflows#restrictions-for-accessing-a-cache , so caching the venv should be fine.

Maybe the key should include the hash of requirements.txt though? As in https://github.com/actions/cache/blob/master/examples.md#simple-example

Reply:
I've rebased that into https://github.com/galaxyproject/galaxy/pull/9170/commits/c76c2322f470b294fcb48a661ff0e9bfda3d07b1